rfcs/0097-no-read-store-dir.md

```diff
@@ -0,0 +1,130 @@
+---
+feature: no-read-store-dir
+start-date: 2021-07-04
+author: Las Safin
+co-authors:
+shepherd-team: @kevincox @7c6f434c @edolstra
+shepherd-leader: @kevincox
```

Comment:
```suggestion
shepherd-leader: @edolstra
```

https://github.com/NixOS/rfcs/pull/97#issuecomment-916076773

Reply:
Didn't notice that the elected shepherd leader had changed... https://github.com/NixOS/rfcs/pull/97#discussion_r701091017

Reply:
@L-as I think we simply missed your yaml assignment when discussing this RFC.